### PR TITLE
App menu search

### PR DESCRIPTION
**Pull request by yogeshgadge**
Adds basic search box for searching Apps

### CODE REVIEW

File: src/app/applications/applications-menu/applications-menu.component.html [hunk 1]
```diff
@@ -23,19 +24,24 @@
...
     </div>
 
-    <md-divider *ngIf="applicationsData.favorites.length > 0 || applicationsData.recents.length > 0"></md-divider>
 
-    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">Refresh
+    <div class="mat-divider" *ngIf="applicationsData.favorites?.length > 0 || applicationsData.recents?.length > 0"></div>
+    <cbp-applications-search></cbp-applications-search>
+
+
+    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">
```
Comment (jscharett) [inline src/app/applications/applications-menu/applications-menu.component.html]: Is this accessible?

File: src/app/applications/applications-menu/applications-menu.component.ts [hunk 2]
```diff
@@ -114,4 +119,9 @@ export class CBPApplicationsMenuComponent implements OnInit, OnDestroy {
 
     get toolbarExpanded(): boolean { return this._isToolbarExpanded; }
 
+
+    // TODO find a way to share this
+    stopPropogation($event: Event) {
```
Comment (jscharett) [inline src/app/applications/applications-menu/applications-menu.component.ts]: Is this supposed to be stopPropogation vs stopPropagation?

File: src/app/applications/applications-menu/applications-menu.component.html (hunk 1 above)
Comment (yogeshgadge) [reply src/app/applications/applications-menu/applications-menu.component.html]: Missed your comments. `<cbp-applications-search></cbp-applications-search>` is probably as accessible as the material components it is composed off.

File: src/app/applications/applications-menu/applications-menu.component.ts (hunk 2 above)
Comment (yogeshgadge) [reply src/app/applications/applications-menu/applications-menu.component.ts]: Typo - it should be `stopPropagation`

**Pull request merged**